content/protocols/index.textile

```diff
@@ -0,0 +1,57 @@
+
+A full list of Ably Client Library SDKs can be found on the "SDK page":https://ably.com/download, and features supported by each SDK can be found in the "features matrix":https://ably.com/download/sdk-feature-support-matrix.
+
+h2(#available-adapters). Available Protocol Adapters
+
+Ably has a number of fully supported adapters:
+
+* MQTT 
+* SSE 
+* AMQP
+* STOMP
+* Pusher
+* PubNub
+
+h3(#mqtt). MQTT
+
+MQTT (MQ Telemetry Transport) is a publish/subscribe, lightweight messaging protocol designed for constrained devices and low-bandwidth networks. One of the major uses of MQTT is with IoT (Internet of Things), where these principles are key to having effective communication between various devices.
+
+MQTT can also be used with Ably as a basic event broker or if we don’t have an SDK for your target platform. However, without an SDK you don’t get access to the full range of platform features and data guarantees.
+
+Read more in the "MQTT section":/protocols/mqtt
+
+h3(#sse). SSE
+
+SSE is a push technology commonly used to send unidirectional message updates or continuous data streams to a browser client. SSE aims to enhance native, cross-browser server-to-client streaming through a JavaScript API called EventSource, standardized as part of HTML5 by the World Wide Web Consortium (W3C).
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime client library, or even an MQTT library, is impractical.
+
+Read more in the "SSE section":/protocols/sse
+
+h3(#amqp). AMQP
+
+The AMQP protocol provides a rich set of functionality to amongst other things bind to exchanges, provision queues and configure routing. The functionality exists so that queues can be dynamically provisioned by clients and messages can be routed to these queues as required.
+
+h3(#stomp). STOMP
+
+STOMP is a simple text-based messaging protocol, typically used for communication between message brokers. It provides an interoperable wire format so that STOMP clients can communicate with any message broker that supports the STOMP protocol and as such is a good fit for use with Ably queues.
+
+h3(#pusher). Pusher
+
+Ably is the only cloud vendor that supports the Pusher protocol. It’s simple to migrate to Ably, or use Ably as a failover for Pusher in hours instead of months. 
+
+Seamlessly migrate from Pusher by connecting your existing clients to the Ably network with practically zero changes to your code.
+
+h3(#pubnub). PubNub
```

Comment:
I'd suggest linking to this [FAQ](https://faqs.ably.com/using-the-ably-pubnub-protocol-adaptor) for now. 

Might also be worth raising a ticket for Pusher/PubNub in this [epic](https://ably.atlassian.net/browse/EDU-1166) to work on improving this content later.